scripts/install

```diff
@@ -190,7 +194,7 @@ ynh_script_progression --message="Starting a systemd service..."
 ###		- And the section "STOP SYSTEMD SERVICE" and "START SYSTEMD SERVICE" in the change_url script
 
 # Start a systemd service
-ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log"
+ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log" --line_match="[INFO] Available at http://:::9000/ in private mode"
```

Comment:
Well uh are you sure it's port 9000 here considering that ynh_find_port earlier is used with port 9009 by default ...?

Also `yunohost firewall allow --no-upnp TCP $port` is probably not needed and in fact trigger a security risk by publicly exposing a service that should only be internally exposed .. (though I didn't check how exactly the app works to be 100% sure about this). 

c.f. https://github.com/YunoHost/issues/issues/1487

Reply:
Fixed by #24